esrally/chart_generator.py

```diff
@@ -1292,6 +1296,83 @@ def ingest(environment, race_config, title):
             },
         }
 
+    @staticmethod
+    def revisions_table(title, environment, race_config):
+        return {
+            "id": str(uuid.uuid4()),
+            "type": "visualization",
+            "attributes": {
+                "title": title,
+                "visState": json.dumps(
+                    {
+                        "title": title,
+                        "type": "table",
+                        "params": {
+                            "perPage": 15,
+                            "sort": {"columnIndex": 0, "direction": "desc"},
+                        },
+                        "aggs": [
+                            {"id": "1", "enabled": True, "type": "count", "params": {}, "schema": "metric"},
+                            {
+                                "id": "2",
+                                "enabled": True,
+                                "type": "date_histogram",
+                                "params": {
+                                    "field": "race-timestamp",
+                                    "interval": "d",
+                                    "customLabel": "day",
+                                },
+                                "schema": "bucket",
+                            },
+                            {
+                                "id": "3",
+                                "enabled": True,
+                                "type": "terms",
+                                "params": {
+                                    "field": "cluster.revision",
+                                    "size": 100,
+                                    "customLabel": "revision",
+                                },
+                                "schema": "bucket",
+                            },
+                            {
+                                "id": "4",
+                                "enabled": True,
+                                "type": "terms",
+                                "params": {
+                                    "field": "cluster.distribution-version",
+                                    "size": 100,
+                                    "customLabel": "version",
+                                },
+                                "schema": "bucket",
+                            },
+                        ],
+                        "listeners": {},
+                    }
+                ),
+                "uiStateJSON": "{}",
+                "description": "revisions",
+                "version": 1,
+                "kibanaSavedObjectMeta": {
+                    "searchSourceJSON": json.dumps(
+                        {
+                            "index": "13969350-badf-11ea-af86-7d06bde52cfd",
```

Comment:
```suggestion
                            "index": str(uuid.uuid4()),
```

Reply:
Oh, I forgot to point that out, I wanted to ask wether there is a way to work around the UUID. It's the id of the `rally-races-*` index pattern on elasticsearch-benchmark-analytics.elastic.co. It's the only way I found to reference the index pattern. For `rally-results-*` this has been solved by changing the id of the index pattern to `rally-results-*` (see for example https://github.com/elastic/rally/blob/8b01f3c853564a6d48337b4d3eb5b81698024be1/esrally/chart_generator.py#L180 ).

One way to resolve this would be to also change the id of the `rally-races-*` index pattern but I'm unsure wether this would have other side effects (or maybe create a copy).

Reply:
That would be nice, and I'm not too worried about side effects. But how can this be done at all? I don't know where to find the ids of index patterns. Or change them.

Reply:
I managed to recreate the index pattern with the `rally-races-*` id by exporting the saved object and re-importing it with the new id. Accordingly, we can use the nicer id here.